tests/python/lib/kphp_server.py

```diff
@@ -145,11 +145,21 @@ def assert_json_log(self, expect, message="Can't wait expected json log", timeou
                 expected_record = expected_records[0]
                 expected_msg = expected_record["msg"]
                 got_msg = json_log_record["msg"]
-                if re.search(expected_msg, got_msg):
+                got_tags = json_log_record.get("tags")
+                if re.search(expected_msg, got_msg) \
+                        and (got_tags.get("process_type") == "general_worker"
+                             or got_tags.get("process_type") == "job_worker"
+                             or got_tags.get("process_type") == "master") \
+                        and (got_tags.get("worker_unique_index") is None or
+                             isinstance(got_tags.get("worker_unique_index"), int)) \
+                        and (isinstance(got_tags.get("pid"), int)):
                     expected_record_copy = expected_record.copy()
                     expected_record_copy["msg"] = ""
                     json_log_record_copy = json_log_record.copy()
                     json_log_record_copy["msg"] = ""
+                    json_log_record_copy["tags"].pop("process_type")
+                    json_log_record_copy["tags"].pop("pid")
+                    json_log_record_copy["tags"].pop("worker_unique_index", "No Key found")
```

Comment:
I think you should do it in another place, somewhere there:
https://github.com/VKCOM/kphp/blob/2c3594f04c432dd262f2cc38efadb7e59fde5501/tests/python/lib/kphp_server.py#L125

Also it's more handy to write like `got_tags.get("process_type") in ['master', 'general_worker', 'job_worker']` instead of multiple `get` and `or`